src/crc32c/__init__.py

```diff
@@ -42,3 +44,57 @@ def value(chunk):
         int: New CRC checksum computed for ``chunk``.
     """
     return crc32c._crc32c_cffi.lib.crc32c_value(chunk, len(chunk))
+
+
+class Checksum(object):
+    """Hashlib-alike helper for CRC32C operations."""
+    __slots__ = ('_crc',)
+
+    def __init__(self, initial_value=b''):
+        """Construct the helper.
+
+        Args:
```

Comment:
Typically we don't put a docstring on `__init__` and put stuff like this in the class docstring